intermediate_source/dynamic_quantization_bert_tutorial.rst

```diff
@@ -528,8 +525,11 @@ We can serialize and save the quantized model for the future use using
 
 .. code:: python
 
-    def ids_tensor(shape, vocab_size, rng=None, name=None):
-        #  Creates a random int32 tensor of the shape within the vocab size
+    import random 
+    global_rng = random.Random()
+
+    def ids_tensor(shape, vocab_size, rng=None,):
```

Comment:
```suggestion
    def ids_tensor(shape, vocab_size, rng=None):
```

Reply:
With the new commit in main branch, the new ids_tensor function is :
**def ids_tensor(shape, vocab_size):
    #  Creates a random int32 tensor of the shape within the vocab size
    return torch.randint(0, vocab_size, shape=shape, dtype=torch.int, device='cpu')**
But I am getting an error which says: _randint() received an invalid combination of arguments - got (int, int, device=str, dtype=torch.dtype, shape=list)_
So do I proceed with the above function or should I use the one I'm currently modifying?